src/Tools/MicroBuild/publish-assets.ps1

```diff
@@ -40,8 +40,10 @@ try
 
     switch ($branchName)
     {
-        "dev15-rc3" { } 
+        "dev15.0.x" { } 
+        "dev15.1.x" { } 
         "master" { } 
+        "dev16" { } 
```

Comment:
In general this change needs to be pair with a change to the `$(RoslynNuGetMoniker)` property

https://github.com/dotnet/roslyn/blob/master/build/Targets/Versions.props#L17

Doing it here means we'll enable publishing on the branches without updating the moniker and hence produce packages that are identical.  Probably need to make this change per branch and pair with the minoker change.